packages/vite/src/runFeServer.ts

```diff
@@ -52,6 +52,8 @@ loadDotEnv({
 // ------------------------------------------------
 
 export async function runFeServer() {
+  const { sendMessage } = await import('execa')
```

Comment:
This is an ESM hack. @dac09 Do you have any recommendations for handling this better?

Reply:
None apart from trying to convert the vite package to ESM (which I'm trying and failing at).

The newer version of execa is also ESM-only, but that would mean all of our packages needing to be ESM first!

Reply:
P.S. From what I could gather I was able to only upgrade this one version of execa, but I may be very wrong about that.

Reply:
Indeed I am wrong. Lame.